lib/srv/alpnproxy/common/protocols.go

```diff
@@ -48,6 +48,10 @@ const (
 	// ProtocolReverseTunnel is TLS ALPN protocol value used to indicate Proxy reversetunnel protocol.
 	ProtocolReverseTunnel Protocol = "teleport-reversetunnel"
 
+	// ProtocolReverseTunnelV2 is TLS ALPN protocol value used to indicate reversetunnel clients
+	// that are aware of proxy peering.
+	ProtocolReverseTunnelV2 Protocol = "teleport-reversetunnelv2"
```

Comment:
Is this used anywhere?

Reply:
Yep used by the agent pool here. https://github.com/gravitational/teleport/blob/f5777287736ff733165ca85c1f6076aaa5ceb9de/lib/reversetunnel/agentpool.go#L442-L455

The purpose is to allow load balancers between the agent and proxy servers to differentiate between agent connections that need to connect to every proxy and those that are using proxy_peering to connect to some subset of proxies.

Reply:
Oh so it's actually not *used* internally by Teleport, I see. Maybe it's worth noting that in a comment?